_includes/docs/latest/resource-coverage.md

```diff
@@ -6,11 +6,14 @@
 | [Billing Accounts](https://cloud.google.com/billing/reference/rest/v1/billingAccounts) |
 | [Buckets](https://cloud.google.com/storage/docs/json_api/v1/buckets#resource) |
 | [Bucket Access Controls](https://cloud.google.com/storage/docs/json_api/v1/bucketAccessControls#resource) |
-| [Cloud IAM GrantableRoles](https://cloud.google.com/iam/reference/rest/v1/roles/queryGrantableRoles) |
-| [Cloud Storage IAM Policies](https://cloud.google.com/storage/docs/access-control/iam-reference) |
 | [Cloud APIs Configuration](https://cloud.google.com/apis/) |
 | [Cloud Audit Logging Configuration](https://cloud.google.com/iap/docs/audit-log-howto) |
+| [Cloud IAM Grantable Roles](https://cloud.google.com/iam/reference/rest/v1/roles/queryGrantableRoles) |
+| [Cloud IAM Organization Roles](https://cloud.google.com/iam/reference/rest/v1/organizations.roles) |
+| [Cloud IAM Project Roles](https://cloud.google.com/iam/reference/rest/v1/projects.roles) |
+| [Cloud Pub/Sub](https://cloud.google.com/pubsub/docs/access-control) |
```

Comment:
We also need to add the IAM policies for pubsub.  But that can be followed up as IAM is not showing up for other resources.